### DISCUSSION TITLE
Help with installation

### DISCUSSION THREAD

**Discussion by ani2796** [goatshriek/stumpless#230] [Q&A]
Hi, I wanted to install Stumpless on my M1 Macbook, but I ran into issues with my PATH variable, and C/C++ compiler from Command Line Tools vs. the one from Homebrew.

I made a more detailed post on [Stack Overflow](https://stackoverflow.com/questions/73428818/wrong-gcc-being-used-macos-monterey-v-12-5). If anyone has any suggestions about what to do, I appreciate the help!

**Comment by goatshriek** [comment]
Thanks for posting! I looked through the writeup and unfortunately don't recognize the problem or have a Mac to test with. I will keep an eye on the thread though, and if there is an issue with Stumpless I will make sure that gets resolved quickly.

**Reply by ani2796** [reply]
Thanks for your reply! Honestly, the issue I'm facing has nothing to do with stumpless, I just wanted to cover my bases in case anyone here may have encountered the same problem in the past. I'll keep an eye out for resolutions too, and thanks again!
